### PR TITLE
Fixes preview for article page by updating dependencies

### LINKED ISSUE #742: [Bug]: Preview for article page does not work
### Contact Details

_No response_

### What happened?

When editing articles page in Sanity, preview does not update live.
Should update preview when making changes

### Reproduction steps

1. Go to sanity studio
2. Select articles page
3. Open preview
4. Make a change (f.ex. title)

### What device are you seeing the problem on?

Desktop

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by fellmirr**
Previews for the articles page was broken
Stemmed from the fact that the next-sanity page could not handle a array::join command in the groq query.
Updating the package solves this issue.
In the update they have split sanity-image and portable text out from the package. Added the required packages and updates some of the code to reflect the changes.

- closes #742

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

**Review by adamalfredsson — APPROVED** [review]
👍

File: lib/config.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ClientConfig } from "next-sanity";
 
-export const config: ClientConfig = {
+export const config = {
```
[the remark this answers is not in the document]
Comment (fellmirr) [reply lib/config.ts:3]: Hmm, I think I did this for some kind of interoperability reason, but seems like the need is no longer there!
Will fix 👍

File: lib/sanity.server.ts
Comment (adamalfredsson) [inline lib/sanity.server.ts:1]: 👍

File: components/shared/responsiveimage.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const ResponsiveImage: React.FC<{
   blur?: boolean;
   urlBuilder?: UseNextSanityImageBuilder;
 }> = ({ image, alt, onClick, priority, layout = "fill", urlBuilder, blur = true }) => {
-  const imageProps = useNextSanityImage({ clientConfig: config }, image, {
+  const imageProps = useNextSanityImage({ clientConfig: projectConfig }, image, {
```
Comment (adamalfredsson) [inline components/shared/responsiveimage.tsx:16]: Are we certain this is safe? For example, in production, the client configuration `useCdn` option is `true`, but undefined in `projectConfig` – could this affect `useNextSanityImage` behaviour?

Comment (fellmirr) [reply components/shared/responsiveimage.tsx]: No, I've switched it out with the sanity client object now, so it should be okay 👍 This is the suggested solution in the documentation.

**Pull request merged**